scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -238,23 +238,23 @@ SG_ Duty_Cycle : 0|32@1- (1,0) [0|1E2] "%" DEBUG
 SG_ Frequency : 32|32@1- (1,0) [0|1E4] "Hz" DEBUG
 
 BO_ 107 BMS_STATE_MACHINE : 1 BMS
-SG_ State : 0|8@1+ (1,0) [0|255] "" DCM
+SG_ State : 0|8@1+ (1,0) [0|255] "" DCM,DIM
 
 BO_ 108 BMS_STATE_OF_CHARGE: 4 BMS
 SG_ State_Of_Charge : 0|32@1- (1,0) [0|1E2] "%" DIM
 
 BO_ 109 BMS_FAULTS: 2 BMS
-SG_ CHARGER_DISCONNECTED_IN_CHARGE_STATE : 0|1@1+ (1,0) [0|1] "" DEBUG
-SG_ CELL_UNDERVOLTAGE_FAULT : 1|1@1+ (1,0) [0|1] "" DEBUG
-SG_ CELL_OVERVOLTAGE_FAULT : 2|1@1+ (1,0) [0|1] "" DEBUG
-SG_ MODULE_COMM_ERROR: 3|1@1+ (1,0) [0|1] "" DEBUG
-SG_ CELL_UNDERTEMP_FAULT: 4|1@1+ (1,0) [0|1] "" DEBUG
-SG_ CELL_OVERTEMP_FAULT: 5|1@1+ (1,0) [0|1] "" DEBUG
-SG_ CHARGER_FAULT_DETECTED : 6|1@1+ (1,0) [0|1] "" DEBUG
-SG_ HAS_REACHED_MAX_V : 7|1@1+ (1,0) [0|1] "" DEBUG
-SG_ CHARGING_EXT_SHUTDOWN_OCCURRED : 8|1@1+ (1,0) [0|1] "" DEBUG
-SG_ TS_OVERCURRENT_FAULT : 9|1@1+ (1,0) [0|1] "" DEBUG
-SG_ PRECHARGE_ERROR : 10|1@1+ (1,0) [0|1] "" DEBUG
+SG_ CHARGER_DISCONNECTED_IN_CHARGE_STATE : 0|1@1+ (1,0) [0|1] "" DIM
```

Comment:
Instead of sending all messages to the DIM, we could send the BMS State to the DIM. I'm if this reduces any bus load but I think it would at least simplify this communication

Reply:
in fact you did that exactly above. So these can change back to DEBUG probably for now. If we decide to implement more descriptive faults we can change this, but I leave that up to the dashboard team.

Reply:
Yup good call, I forgot to switch these back.